### PR TITLE
runtime: prefs: use GR logging for debug messages

### PR DESCRIPTION
**Pull request by sashko**
Part of #3362.

### CODE REVIEW

**Comment by drmpeg** [comment]
It's not really useful to modify this file. It was never executed and it's no longer being built. It's just there for historical purposes. It's been replaced with a pre-built file so that gr-dtv can be cross compiled. See:

https://github.com/gnuradio/gnuradio/commit/5af80cc3146a908f3d3fd4fb201f8eebe03eff1e

**Comment by sashko** [comment]
> It's not really useful to modify this file. It was never executed (during runtime) and it's no longer being built. It's just there for historical purposes. It's been replaced with a pre-built file so that gr-dtv can be cross compiled. See:
> 
> [5af80cc](https://github.com/gnuradio/gnuradio/commit/5af80cc3146a908f3d3fd4fb201f8eebe03eff1e)

Understood. Let's try with something else then...

File: gnuradio-runtime/lib/prefs.cc [hunk 1]
```diff
@@ -72,6 +73,10 @@ std::vector<std::string> prefs::_sys_prefs_filenames()
 
 void prefs::_read_files(const std::vector<std::string>& filenames)
 {
+    gr::logger_ptr logger, debug_logger;
+    gr::configure_default_loggers(logger, debug_logger, "prefs");
+    std::ostringstream msg;
```
Comment (willcode) [inline gnuradio-runtime/lib/prefs.cc:78]: msg never gets logged.

File: gnuradio-runtime/lib/prefs.cc [hunk 2]
```diff
@@ -98,14 +103,14 @@ void prefs::_read_files(const std::vector<std::string>& filenames)
...
+                msg << "WARNING: Config file '" << fname
+                    << "' failed to parse:" << std::endl;
+                msg << e.what() << std::endl;
+                msg << "Skipping it" << std::endl;
             }
         } else { // infile.good();
-            std::cerr << "WARNING: Config file '" << fname
-                      << "' could not be opened for reading." << std::endl;
+            msg << "WARNING: Config file '" << fname
+                << "' could not be opened for reading." << std::endl;
```
Comment (marcusmueller) [inline gnuradio-runtime/lib/prefs.cc:113]: This is in the right spirit, but `msg` will have to end up in the logger, too!

**Review by marcusmueller — CHANGES_REQUESTED** [review]

**Comment by mbr0wn** [comment]
@sashko We would still like to merge this. It's a small enough change, and a good one. However, the comments need to be resolved (i.e., `msg` needs to be logged), and can you please sign-off your commit (`git commit -s`). We'll close this if it doesn't get resolved, to reduce clutter in our PR queue.

**Comment by mbr0wn** [comment]
Closing due to lack of action (we need to de-clutter the PR queue). Please reopen if you can update the PR. Thanks for your contribution!

**Pull request closed without merging**